src/routes/docs/homepage.md

```diff
@@ -16,16 +16,20 @@
 
 ## Summary
 
-This Web Login Framework is a development tool set for easily developing an alternative to the ForgeRock-hosted login application or building your own login application or component from scratch. The first product to be provided by this framework is the Login Widget which is a JavaScript component that can be installed into any modern JavaScript application. A future product provided by this framework will be a self-contained Login Application that can be self-hosted which will be useful as a centralized login app to your ecosystem of applications.
+The Login Widget produced by this framework is intended to be an all-inclusive, UI component that can be installed with any modern JavaScript app for handling the default login, registration and related user flows. It can be used within a React, Vue, Angular or any other modern JavaScript framework (does not currently support Node.js or server-rendering (SSR)).
```

Comment:
> The Login Widget produced by this framework is intended to be an all-inclusive

I know you've mentioned this terminology before, so I feel bad admitting i don't fully understand this, but maybe we aren't at the current maturity of development.

I think this is confusing because there isn't a clear delineation between what is "widget" and what is "framework". Is this something thats worth documenting? What is the delineation between framework and widget?

Reply:
I've added additional documentation around this to the PR. The easiest way to look at it is the widget is what we publish to NPM (the product) and the framework is what you download locally to modify and rebuild the widget. The most important aspect to this framework is that it will be eventually used to produce more than just the widget. We plan to produce a standalone Login App, and/or a separately deployable BFF as well.

It will look more like a framework once we implement the ability to write custom Svelte components that can overwrite the default components, allowing developers to customize not only the look and feel, but also the structure and behavior of the authentication forms, through a strict set of patterns, guidelines and APIs.